### ISSUE TITLE
No miners in panel

### ISSUE THREAD

**Issue by Serasar** [UnamSanctam/UnamWebPanel#242]
No miners shown in panel and I don't know why.
Checked the following:
- DB is writable
- Miner is running and endpoint is correct
- Tried older Version 1.7.0
I suspect it may be the bug in endpoint.php. Is there a way to debug it?

**Comment by shainodev** [comment]
do u mean you opened it on other peoples pcs or u tested it on ur own?

**Comment by UnamSanctam** [comment]
Make sure you're using the correct http/https. And also try making/saving a configuration inside the "Configurations" tab just to make sure it's writable.

**Comment by Serasar** [comment]
I've checked writability using "Configurations" tab. Endpoint have correct https

**Comment by Serasar** [comment]
Yes cert from let's encrypt, checked, it's working. PHP is 8.1 Strange error in apache error.log:
[Tue Feb 21 12:24:53.148697 2023] [php:error] [pid 11585] [client hide_ip:62537] script '/var/www/html/xmlrpc.php' not found or unable to stat

**Comment by UnamSanctam** [comment]
Make sure you don't have any default Apache example files or anything left. But check your Apache access log (or enable it if disabled), you should then be able to see there if the miner is able to reach it. And you also have https entered in the "API Endpoint URL" inside the miner?

**Comment by Serasar** [comment]
> And you also have https entered in the "API Endpoint URL" inside the miner?
Yes I do have it entered
--cinit-api="https://mydomain.tk/api/endpoint.php"

**Comment by Serasar** [comment]
Turned it out it didn't want to send pings from my OS. Tried it on other windows and it worked

**Comment by neoandersen** [comment]
I have a suggestion, it would be useful to integrate adding a link for iploger, it is useful to track the work of the builds. And clipper integration for address spoofing would be good.

**Comment by UnamSanctam** [comment]
You can add a link for an IPLogger inside the "Remote Configuration" though that will call it every 100 minutes which could be a downside. I probably won't add a clipper, I did finish a UnamCrypoClipper project but I haven't released it because I was asked not to by quite a few people due to its negative effects on crypto.

**Comment by neoandersen** [comment]
> You can add a link for an IPLogger inside the "Remote Configuration" though that will call it every 100 minutes which could be a downside. I probably won't add a clipper, I did finish a UnamCrypoClipper project but I haven't released it because I was asked not to by quite a few people due to its negative effects on crypto.

In principle, IPLoger is not important. Incoming streams will be visible anyway.
